geoportailv3/templates/index.html

```diff
@@ -46,7 +46,12 @@
           </div>
         </div>
 
-        <div class="offline-msg alert-danger" translate>You are currently offline.</div>
+        <div class="offline-msg alert-danger" translate>
+          <span ng-if="!mainCtrl.offlineMode.isEnabled()">You are currently disconnect.
```

Comment:
Should be worded `You are currently disconnected.`.
In addition, you should test the network status, not the offline mode status.

Reply:
Well a message is displayed if we are offline or disconnected, so there is already a test uppon in the html tree (applied with the css).
Testing here if we are not offline (like in the PR) means we are here because it is disconnected.
Otherwise i should have a double test : disconnected AND not offline mode.